js/myparcel.js

```diff
@@ -302,63 +324,60 @@ MyParcel = {
              */
             if ($('#mypa-pickup-express-selector').prop('checked')) {
                 $('#s_method_myparcel_pickup_express').click();
-                MyParcel.addPickupToExternalInput('retailexpress');
+                MyParcel.addPickupToExternalInput(MyParcel.DELIVERY_PICKUP_EXPRESS);
                 MyParcel.addStyleToPrice('#mypa-pickup-express-price');
                 return;
             } else {
                 MyParcel.addStyleToPrice('#mypa-pickup-price');
             }
 
-
             $('#s_method_myparcel_pickup').click();
-            MyParcel.addPickupToExternalInput('retail');
+            MyParcel.addPickupToExternalInput(MyParcel.DELIVERY_PICKUP);
         }
     },
 
-    addPickupToExternalInput: function (selectedPriceComment) {
+    addPickupToExternalInput: function(selectedPriceComment) {
         var locationId = $('#mypa-pickup-location').val();
         var currentLocation = MyParcel.getPickupByLocationId(MyParcel.storeDeliveryOptions.data.pickup, locationId);
 
-        var result = jQuery.extend({}, currentLocation);
+        var result = $.extend({}, currentLocation);
```

Comment:
Je moet jQuery laten staan. Magento 1 gebruikt prototypejs. Daar kan je ook $ gebruiken. Daarom kan je niet $ gebruiken. Dit moet je overal in dit bestand terugzetten.